lighthouse-core/gather/gatherers/trace-compat.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview
+ * This gatherer remaps the result of the Trace gatherer for compatibility with legacy Lighthouse
+ * when devtools logs and traces were special-cased.
+ */
+
+const TraceGatherer = require('./trace.js');
+const FRGatherer = require('../../fraggle-rock/gather/base-gatherer.js');
+
+/** @implements {LH.Gatherer.FRGathererInstance<'Trace'>} */
+class TraceCompat extends FRGatherer {
+  /** @type {LH.Gatherer.GathererMeta<'Trace'>} */
+  meta = {
+    supportedModes: ['navigation'],
```

Comment:
this is actually a fairly important signal that we shouldn't try to compute all the metrics over an arbitrary timespan

longer term I think this evolves into a `Trace` and `NavigationTrace` artifact or something where the latter is guaranteed to have been collected over a navigation